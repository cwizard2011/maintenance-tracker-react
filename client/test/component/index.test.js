import React from 'react';
import { shallow } from 'enzyme';
import App from '../../src/components/index';

describe('Guest Navigation', () => {
  it('render guest navigation correctly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
