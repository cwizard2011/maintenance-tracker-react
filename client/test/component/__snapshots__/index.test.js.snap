// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Guest Navigation render guest navigation correctly 1`] = `
<div>
  <Connect(Home) />
</div>
`;
